index.rst

```diff
@@ -0,0 +1,190 @@
+
+1.  The analytic approach
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If (e1,e2) is the measure of the galaxy ellipticity and (g1,g2) is the applied shear, the deviation of (e1-g1,e2-g2) from (0,0) is a measure of our error.  Of course, this measurement is dominated by shape noise. By averaging e1 and e2 over and entire subfield, we get an estimator of the applied shear, as well as the error in this estimation.
+
+Since the regression to determine m1, c1, m2 and c2 uses as its data the (e1avg, e2avg)  vs. (g1, g2), we can use the errors in e1avg and e2avg to determine the errors in the regression values.  See Numerical Recipes section 15.2 for a description of this calculation.
+
+2.  Bootstrap estimation
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+However, since we did not entirely trust the analytic approach, we attempted to determine the error in our shear bias calculation by running our entire simulation 100-300 times using a bootstrap technique.  That is, we did each subfield measurement again (for each of the 200 subfields) with 5000 galaxy pairs, sampled from the original 5000 galaxy pairs with replacement.  The bootstrap method consistently gives an error which is larger than the analytic approach by a factor of 2.1-2.3.
+
+Since the bootstrap estimation indicates a larger error, we decided to increase our errors uniformly by a factor of 2.2 for this study.  It would obviously be even better to examine the errors over a large number of simulations.  20 simulations with 2 million galaxies each would probably give us a better handle on both the error in the shear biases, and also an absolute calibration of the shear measurements.
+
+
+
+Testing Parameterizations of ShapeletPsfApprox (DM-1136 and DM-4214)
+========================
+
+These are a set of tests which are intended to test ShapeletPsfApprox(SPA), and in particular, to find the shapelet order above which there is little additional value.  SPA is a four component “multi-shapelet function” model of the Psf, where the four components from narrowest to broadest are named “inner”, “primary”, “wings”, and “outer”.  An SPA model is designated in this document with four numbers:  for example, 1234 means inner order = 1, primary order = 2, wings order = 3, and outer order = 4.
+
+Test setup
+--------------
+
+This set of tests is run using the great3sims configuration of 200 subfields with 10000 galaxies each.  All were run with a single fixed Psf with filter f3 and raw seeing 0.7 arcsec.  The errors shown below are estimated using bootstrap resampling, as discussed in the introduction.  Please note that most of the differences in parameterizations were washed out when the errors were increased by x2.2.  So getting a better estimate of the errors could be important to these results.
```

Comment:
I'm unclear here if you're _both_ bootstrapping and increasing the errors by a factor of 2.2, and which combination of the above causes the errors to "wash out".